.circleci/config.yml

```
@@ -469,7 +476,7 @@ workflows:
       - simulate_sequence:
           name: simulate_sequence_eth
           network: "eth"
-          tasks: "030"
+          tasks: "unichain-001"
           block_number: "21938584" # If not specified, the latest block number is used.
```

We need to remove the block number to make sure we are at the last block used.
```suggestion
          block_number: "" # If not specified, the latest block number is used.
```